t/config.t

```diff
@@ -201,6 +201,335 @@ subtest 'Everything but NoWarnings' => sub {
     }
     qr/DB\.engine.*Excel/, 'die: Invalid DB.engine value';
 
+    throws_ok {
+        my $text = q{
+            [DB]
+            engine = SQLite
+            polling_interval = hourly
+
+            [SQLITE]
+            databse_file = /var/db/zonemaster.sqlite
+        };
+        Zonemaster::Backend::Config->parse( $text );
+    }
+    qr{DB\.polling_interval.*hourly}, 'die: Invalid DB.polling_inteval value';
```

Comment:
typo `DB.polling_inteval` -> `DB.polling_interval`

Reply:
You do have a good eye. Also fixed.